core/templates/services/state-interaction-stats.service.ts

```diff
@@ -111,24 +106,26 @@ export class StateInteractionStatsService {
           exploration_id: explorationId,
           state_name: state.name,
         }))
-      .toPromise().then(response => <IStateRulesStats>{
-        exploration_id: explorationId,
-        state_name: state.name,
-        visualizations_info: response.visualizations_info.map(info => ({
-          addressed_info_is_supported: info.addressed_info_is_supported,
-          data: info.data.map(datum => <IAnswerData>{
-            answer: this.getReadableAnswerString(state, datum.answer),
-            frequency: datum.frequency,
-            is_addressed: (
-              info.addressed_info_is_supported ?
+      .toPromise().then(response => {
```

Comment:
Should the typedef be removed here? /cc @nishantwrp

Reply:
I had been the one to add the typedef, it's no longer necessary since the type can be derived by the return value (which is now `this.cachedStats`, typed as `IStateRulesStats`).